src/services/requests/home/index.ts

```diff
@@ -307,7 +307,7 @@ const homeKlvChartCall = async (
 const homeKfiDataCall = async (): Promise<IGeckoResponse | undefined> => {
   try {
     const res = await api.get({
-      route: 'coins/klever-finance',
+      route: 'coins/klever.org',
```

Comment:
i think don't need to change in those lines

Reply:
you are correct, only the old one works